### PR TITLE
Fix command line names and documentation

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Comment by harshavardhana** [comment]
all the changes reviewed by @abperiasamy - these are all breaking changes.

File: cmd/admin-policy-list.go [hunk 1]
```diff
@@ -42,24 +42,24 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-  1. List all policies.
+  1. List all policy.
```
Comment (ebozduman) [inline cmd/admin-policy-list.go]: Inconsistent with `Usage: list all policies`
Either `List all policy` should be in plural or `Usage:` needs to be singular.

File: cmd/admin-profile-start.go [hunk 2]
```diff
@@ -25,20 +25,20 @@ import (
...
 		Value: "mem",
 	},
 }
 
-var adminProfilingStartCmd = cli.Command{
+var adminProfileStartCmd = cli.Command{
 	Name:            "start",
-	Usage:           "Start recording profiling data",
-	Action:          mainAdminProfilingStart,
+	Usage:           "Start recording profile data",
```
Comment (ebozduman) [inline cmd/admin-profile-start.go]: In previous `Usage:`s, everything was in lower case...
`Usage:  "Start recording profile data",` => `Usage:  "start recording profile data",`

File: cmd/admin-profile-stop.go [hunk 3]
```diff
@@ -27,10 +27,10 @@ import (
 	"github.com/minio/mc/pkg/probe"
 )
 
-var adminProfilingStopCmd = cli.Command{
+var adminProfileStopCmd = cli.Command{
 	Name:            "stop",
-	Usage:           "Stop and download profiling data",
-	Action:          mainAdminProfilingStop,
+	Usage:           "Stop and download profile data",
```
Comment (ebozduman) [inline cmd/admin-profile-stop.go]: In previous `Usage:`s, everything was in lower case...
`Usage:  "Stop and download profiling data",` => `Usage:  "stop and download profiling data",`

**Comment by harshavardhana** [comment]
@ebozduman one request is always review on latest commit, reviewing on older commits is harder for me to look at. The best option is always refresh by clicking the "refresh" button presented by Github.

File: cmd/rm-main.go [hunk 4]
```diff
@@ -39,47 +39,47 @@ var (
...
 		},
 		cli.IntFlag{
 			Name:  "newer-than",
-			Usage: "Remove objects newer than N days",
+			Usage: "remove objects newer than N days",
 		},
 		cli.StringFlag{
 			Name:  "encrypt-key",
-			Usage: "Encrypt object (using server-side encryption)",
+			Usage: "encrypt object (using server-side encryption)",
```
[the remark this answers is not in the document]
Comment (harshavardhana) [reply cmd/rm-main.go]: yes encryption key is needed to remove the object

File: cmd/mirror-main.go [hunk 5]
```diff
@@ -39,61 +39,61 @@ var (
...
 		},
 		cli.IntFlag{
 			Name:  "newer-than",
-			Usage: "Select objects newer than N days",
+			Usage: "select objects newer than N days",
 		},
 		cli.StringFlag{
 			Name:  "storage-class, sc",
-			Usage: "Set storage class for object",
+			Usage: "set storage class for object",
```
[the remark this answers is not in the document]
Comment (harshavardhana) [reply cmd/mirror-main.go]: no select @ebozduman

File: cmd/config-main.go [hunk 6]
```diff
@@ -35,7 +35,7 @@ var (
 
 var configCmd = cli.Command{
 	Name:            "config",
-	Usage:           "Manage mc configuration file.",
+	Usage:           "configure minio client",
```
Comment (ebozduman) [inline cmd/config-main.go]: It looks like you've started capitalizing `minio` => `Minio`

File: cmd/mb-main.go [hunk 7]
```diff
@@ -68,7 +68,7 @@ EXAMPLES:
    5. Create a new directory including its missing parents (equivalent to 'mkdir -p').
       $ {{.HelpName}} /tmp/this/new/dir1
 
-   6. Create multiple directories including its missing parents (behavior similar to 'mkdir -p').
+   6. Create multiple director2ies including its missing parents (behavior similar to 'mkdir -p').
```
Comment (ebozduman) [inline cmd/mb-main.go]: Typo
`director2ies` => `directories`

File: cmd/watch-main.go [hunk 8]
```diff
@@ -36,26 +36,26 @@ var (
 		cli.StringFlag{
 			Name:  "events",
 			Value: "put,delete,get",
-			Usage: "Filter specific types of events. Defaults to all events by default.",
+			Usage: "filter specific types of events. defaults to all events by default",
```
Comment (ebozduman) [inline cmd/watch-main.go]: `... events. defaults...` => `... events; defaults...`

File: docs/minio-admin-complete-guide.md [hunk 9]
```diff
@@ -3,13 +3,13 @@
...
 
 ```sh
-service      Stop, restart or get status of Minio server
-info         Display Minio server information
-users        Manage regular users
-policies     Manage canned policies
-credentials  Change Admin server access and secret keys
-config       Manage configuration file
-heal         Heal disks, buckets and objects on Minio server
+service      stop, restart or get status of minio server
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 10]
```diff
@@ -3,13 +3,13 @@
...
+info         display minio server information
+user         manage users
+policy       manage canned policies
+credential   change admin server access and secret keys
+config       manage configuration file
+heal         heal disks, buckets and objects on minio server
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 11]
```diff
@@ -213,29 +213,29 @@ Skip SSL certificate verification.
...
 |   |
 |:---|
-|[**service** - Start, stop or get the status of Minio server](#service) |
-|[**info** - Display Minio server information](#info) |
-|[**users** - Manage users](#users) |
-|[**policies** - Manage policies](#policies) |
-|[**credentials** - Change **admin** server access and secret keys](#credentials) |
-|[**config** - Manage server configuration file](#config)|
-|[**heal** - Heal disks, buckets and objects on Minio server](#heal) |
+|[**service** - start, stop or get the status of minio server](#service) |
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 12]
```diff
@@ -213,29 +213,29 @@ Skip SSL certificate verification.
...
+|[**info** - display minio server information](#info) |
+|[**user** - manage users](#user) |
+|[**policy** - manage canned policies](#policy) |
+|[**credential** - change **admin** server access and secret keys](#credential) |
+|[**config** - manage server configuration file](#config)|
+|[**heal** - heal disks, buckets and objects on minio server](#heal) |
 
 <a name="service"></a>
-### Command `service` - Stop, restart or get status of Minio server
+### Command `service` - stop, restart or get status of minio server
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 13]
```diff
@@ -213,29 +213,29 @@ Skip SSL certificate verification.
...
 `service` command provides a way to restart, stop one or get the status of Minio servers (distributed cluster)
 
 ```sh
 NAME:
-  mc admin service - Stop, restart or get status of Minio server
+  mc admin service - stop, restart or get status of minio server
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 14]
```diff
@@ -213,29 +213,29 @@ Skip SSL certificate verification.
...
 
 FLAGS:
-  --help, -h                       Show help.
+  --help, -h                       show help
 
 COMMANDS:
-  status   Get the status of Minio server
-  restart  Restart Minio server
-  stop     Stop Minio server
+  status   get the status of minio server
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Next 3 capitalizations
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 15]
```diff
@@ -260,10 +260,10 @@ Restarted `play` successfully.
 
 ```sh
 NAME:
-  mc admin info - Get Minio server information
+  mc admin info - get minio server information
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

File: docs/minio-admin-complete-guide.md [hunk 16]
```diff
@@ -315,79 +315,79 @@ mc admin policies list --json myminio/
 {"status":"success","policy":"newpolicy"}
 ```
 
-<a name="users"></a>
-### Command `users` - Manage regular users
-`users` command to add, remove, enable, disable, list users on Minio server.
+<a name="user"></a>
+### Command `user` - Manage users
+`user` command to add, remove, enable, disable, list users on minio server.
```
Comment (ebozduman) [inline docs/minio-admin-complete-guide.md]: Capitalization 
`minio` => `Minio`

**Comment by deekoder** [comment]
We need to remember that this is a breaking change. When we make the next release, we need to do this. @harshavardhana the mc complete guide and mc quickstart guide also needs to be updated.

**Pull request merged**